core/sequencer/sequencer.go

```diff
@@ -121,16 +121,31 @@ func (s *Sequencer) ForAllMasterships(ctx context.Context, f func(ctx context.Co
 	return nil
 }
 
-// RunBatchForAllMasterships runs KeyTransparencySequencerClient.RunBatch on all
-// directories this sequencer is currently master for.
-func (s *Sequencer) RunBatchForAllMasterships(ctx context.Context, batchSize int32) error {
-	glog.Infof("RunBatchForAllMasterships")
+// DefineRevisionsForAllMasterships runs KeyTransparencySequencerClient's
+// DefineRevisions method on all directories that this sequencer is currently
+// master for.
+func (s *Sequencer) DefineRevisionsForAllMasterships(ctx context.Context, batchSize int32) error {
```

Comment:
I think there's a more generic RunForAllMasterships function that might reduce some code duplication